java/sage/Wizard.java

```diff
@@ -15,66 +15,34 @@
  */
 package sage;
 
-import static sage.SageConstants.LITE;
-
 import org.apache.lucene.analysis.Analyzer;
 import org.apache.lucene.document.Document;
 import org.apache.lucene.document.Field;
-import org.apache.lucene.index.CorruptIndexException;
-import org.apache.lucene.index.IndexCommit;
-import org.apache.lucene.index.IndexReader;
-import org.apache.lucene.index.IndexWriter;
-import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.index.*;
```

Comment:
My personal preference is to never import .*. I've been bitten by that before where it was using the wrong class because I wasn't aware something with the same class name was in a .* imported package.